source/2020-05-08-the-ember-times-issue-147.md

```diff
@@ -37,25 +39,17 @@ Check out the full list of features and improvements over at the project's [chan
 
 ---
 
-## [Section title in sentence case 🐹](section-url)
+## [Auto save en Ember Octane 💾](https://betocantu93.com/autosave-in-ember-octane/)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
-<please include link to external article/repo/etc in paragraph / body text, not just header title above>
+[Alberto Cantú (@betocantu93)](https://github.com/betocantu93), in his first blog post in Spanish, explores [how to implement autosave in Ember](https://betocantu93.com/autosave-in-ember-octane/). We encourage you to check it out! Alberto starts with a basic form with a submit button, then introduces autosave and addresses potential problems using `ember-concurrency`.
 
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+Do you follow a similar pattern or have another approach to autosaving your user's form? Share your ideas with the JS community today!
 
 ---
 
-## [Section title in sentence case 🐹](section-url)
+## [Embroider & memoization RFCs entered final comment period 💬](https://github.com/emberjs/rfcs/pull/507)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
-<please include link to external article/repo/etc in paragraph / body text, not just header title above>
-
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+In case you missed, the RFCs for [Embroider](https://github.com/emberjs/rfcs/pull/507) and [memoizing tracked properties](https://github.com/emberjs/rfcs/pull/615) entered their final comment period (FCP) on May 1.
```

Comment:
I don't know if an FCP can last for more than 7 days. I assumed the due dates for these RFCs are today and wasn't sure what more I could talk about. 😓

Reply:
Yeah, I agree, I also assume that the comment period will close very soon, so I think this wording works great!